javascript/javascript1/week1/Age-ify2.js

```diff
@@ -0,0 +1,8 @@
+//trying to calculate my age- storing the heard codded value variables//
+let yearOfBirth = 1993;
+let yearFuture = 2045;
+let demo1 = " You will be "
+let demo2 = " years old in 2045 "
```

Comment:
I think it will be easier to use the string `You will be` directly with `console.log` without storing it in a variable. The same for `demo2`